android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -202,6 +203,7 @@ object ComponentsDependencies {
     const val thirdparty_sentry_latest = "io.sentry:sentry-android:${Versions.sentry_latest}"
     const val thirdparty_zxing = "com.google.zxing:core:${Versions.zxing}"
     const val thirdparty_disklrucache = "com.jakewharton:disklrucache:${Versions.disklrucache}"
+    const val thirdparty_coil_svg = "io.coil-kt:coil-svg:${Versions.coil}"
```

Comment:
Drive-by 🚗 : Can you add a nice big ol' comment above here that we are using this for the decoder only? It comes with an OkHttp client that we can't swap out with different http client, so we aren't using that part of the code. 

"DO NOT USE ICON FETCHING FROM COIL."

It's good that you are adding it to the browser-icons library so that it protects us from misusing it in other components or apps, but not everyone diligently looks at git blames and may end up seeing this line here, and then thinking it's okay to add it to other places.

Reply:
@jonalmeida good suggestion, added the suggested comment

Reply:
Sorry, that was a bad use of quotes on my side.

I was trying to convey the sentiment in that message, but it would be good to write out a blurb on the why. That's the thing that the code is not able to convey.

Reply:
Gotcha! Apologies, I took that quite literally.
I've updated the comment here and added a reference to it in `SvgIconDecoder`

Reply:
We could discourage using other code parts than this one by using detekt rules. ( a custom rule or a generic one like `ForbiddenImport` or `ForbiddenMethodCall`).
That could be a follow-up if we decide on what specific methods we want to discourage.

Reply:
That's a great suggestion @mcarare. I will make that a new issue, unless there are any objections to this?

Reply:
No issues on my end.